external/CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ if (VERONA_DOWNLOAD_LLVM)
     USES_TERMINAL_DOWNLOAD true
   )
 else()
+  separate_arguments(LLVM_EXTRA_CMAKE_ARGS UNIX_COMMAND ${LLVM_EXTRA_CMAKE_ARGS})

```

Comment:
I'm not very familiar with cmake, but I think this was a legitimate bug with this spec.  Without this, whatever I would set via `-DLLVM_EXTRACMAKE_ARGS="..."` wasn't getting propagated to the cmake command used to configure the LLVM build.

Reply:
We had that problem before, where some high-level flags weren't propagating to the sub-projects. @mjp41 knows more how that works.

Reply:
I thought it was fixed, but it is untested.  Was it broken for a single additional argument, or only for multiple.

Reply:
Only for multiple.